### PR TITLE
User experience

### PR DESCRIPTION
**Pull request by AsalaKM**
**Edit User Experience**     **&**     **Add master style**
Adding a new Routs and links to make the user experience good in the site.
Adding master styles that we needed in the site

**Relates #55**

### CODE REVIEW

**Comment by FarahZaqout** [comment]
1- pull requests should always relate to issues.
2 - I still don't understand what the pull request does exactly. User experience where? What was the problem or what did we solve?

**Comment by FarahZaqout** [comment]
Resolve conflicts.

**Review by SalwaMugh — APPROVED** [review]

File: src/controllers/supportDetails.js [hunk 1]
```diff
@@ -20,9 +20,7 @@ exports.get = (req, res, next) => {
 exports.put = (req, res, next) => {
   const data = req.body;
   updateTicket(data)
-    .then((response) => {
-      res.send(data);
-    })
+    .then(res.send(data))
```
Comment (FarahZaqout) [inline src/controllers/supportDetails.js]: this needs to be a callback.
inside the ```.then()``` will always fire if it's not inside a callback. Even if it's an error.

File: src/model/queries/ticket.js [hunk 2]
```diff
@@ -40,8 +48,16 @@ const getSolvedCount = () => {
 
 const getNotSolvedCount = () => {
   const sql = {
-    text: 'SELECT COUNT(ticket_no) FROM ticket WHERE status_type LIKE $1',
-    values: ['not_solved'],
+    text: 'SELECT COUNT(ticket_no) from ticket Where status_type NOT like $1',
```
Comment (FarahZaqout) [inline src/model/queries/ticket.js]: capitalize sql statements.

File: src/views/homeManager.hbs [hunk 3]
```diff
@@ -1,20 +1,82 @@
...
-    <h2 class="page-title" >الرئيسية</h2>
-    <div class="ticket-view">
-        <h3> بطاقات مفتوحة: {{#each tNotSolved}}<span>{{count}}</span>{{/each}}</h3>
-         <h3>بطاقات مغلقة :{{#each tSolved}}<span>{{count}}</span>{{/each}}</h3>
-        <h3>المجموع:{{#each tCount}}<span>{{count}}</span>{{/each}}</h3>
+<h2 class="page-title" >الرئيسية</h2>
+<!-- START  STATIC INFO  -->
+    <div class="top">
+        <div class="boxes">
+            <a href="/closed/not">
```
Comment (FarahZaqout) [inline src/views/homeManager.hbs]: ```/closed/not/```
```/solved/solved``` do not make much sense

**Review by FarahZaqout — CHANGES_REQUESTED** [review]

**Review by FarahZaqout — APPROVED** [review]

**Pull request merged**